Editor/ManifestGenerator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.IO;
+using System.Text;
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+
+namespace Elevator89.BuildPresetter
+{
+    public class ManifestGenerator : IPreprocessBuildWithReport
+    {
+        public int callbackOrder => 0;
+
+        public void OnPreprocessBuild(BuildReport report)
+        {
+            if (report.summary.platform == BuildTarget.WebGL || report.summary.platform == BuildTarget.Android)
+            {
+                string streamingAssetsPath = Application.dataPath + "/StreamingAssets";
+                string outputPath = streamingAssetsPath + "/manifest.txt";
+
+                StringBuilder manifestContent = new StringBuilder();
+                string[] files = Directory.GetFiles(streamingAssetsPath, "*", SearchOption.AllDirectories)
+                    .Where(file => !file.EndsWith(".meta"));
+
+                foreach (string file in files)
+                {
+                    string relativePath = file.Substring(streamingAssetsPath.Length + 1).Replace("\\", "/");
+                    manifestContent.AppendLine(relativePath);
+                }
+
+                File.WriteAllText(manifestFilePath, manifestContent.ToString());
```

Comment:
We also need to delete created file, otherwise it will fill both `StreamingAssets` and `~StreamingAssets` (excluded folder), leading to an exception on the next build attempt

Reply:
исправила